src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -757,10 +757,15 @@ public partial class UdpClient : System.IDisposable
         public byte[] Receive([System.Diagnostics.CodeAnalysis.NotNullAttribute] ref System.Net.IPEndPoint? remoteEP) { throw null; }
         public System.Threading.Tasks.Task<System.Net.Sockets.UdpReceiveResult> ReceiveAsync() { throw null; }
         public int Send(byte[] dgram, int bytes) { throw null; }
+        public int Send(System.ReadOnlySpan<byte> datagram) {throw null; }
         public int Send(byte[] dgram, int bytes, System.Net.IPEndPoint? endPoint) { throw null; }
+        public int Send(System.ReadOnlySpan<byte> datagram, System.Net.IPEndPoint? endPoint) { throw null; }
```

Comment:
endPoint should not be nullable

Reply:
similar for other APIs below, and for hostname below.

Reply:
Actually, nevermind. It looks like you are doing the right thing here in matching the existing APIs. The problem is the API approval got the nullability wrong here. I'll follow up on that.